wiki/Tournaments/RDOC/2019/en.md

```diff
@@ -16,7 +16,6 @@ The **Royal Danish osu!Cup 2019** (***RDOC 2019***) is an ongoing Danish 1v1 dou
 | --: | :-- |
 | Registration phase | 2020-01-10/2020-01-24 |
 | Qualifiers | 2020-02-07/2020-02-16 |
-| Live drawings | *to be determined* |
```

Comment:
didn't happen?

Reply:
nope, since the format was changed from groupstage to ro32 there needs to be no live drawings